io/subsystem/src/main/java/org/wildfly/extension/io/IOExtension.java

```diff
@@ -69,7 +69,7 @@ public void initializeParsers(ExtensionParsingContext context) {
 
     @Override
     public void initialize(ExtensionContext context) {
-        final SubsystemRegistration subsystem = context.registerSubsystem(SUBSYSTEM_NAME, ModelVersion.create(4));
+        final SubsystemRegistration subsystem = context.registerSubsystem(SUBSYSTEM_NAME, ModelVersion.create(4, 1));
```

Comment:
Do I need to up the model version because I have deprecated a resource? I am assuming yes, although I am not sure if it should be major minor or micro.

Reply:
@stuartwdouglas It was already bumped for WF Core 4 as part of https://issues.jboss.org/browse/WFCORE-3333

Reply:
In this case it should stay at 4.0.
unless it doesn't get merged for wildfly-core 4.0, in that case it needs to be 5.0 (and few other PRs as well)